omim2obo/main.py

```diff
@@ -198,36 +239,49 @@ def omim2obo(use_cache: bool = False):
             gene_label_err = 'Warning: Only 1 symbol picked for label for gene term, but there were 2 to choose' \
                  f'from. Unsure which is best. Picking the first.\nhttps://omim.org/entry/{omim_id} - {pref_symbols}'
             if len(pref_symbols) > 1:
-                LOG.warning(gene_label_err)  # todo: decide the best way to handle these situations
+                LOG.warning(gene_label_err)  # todo: rare (n=1?), but decide the best way to handle these situations
             graph.add((omim_uri, RDFS.label, Literal(pref_symbols[0])))
         else:
             graph.add((omim_uri, RDFS.label, Literal(label_cleaner.clean(pref_title))))
 
         # todo: .clean()/.cleanup_label() 2nd param `explicit_abbrev` should be List[str] instead of str. And below,
         #  should pass all symbols/abbrevs from each of preferred, alt, included each time it is called. If no symbols
         #  for given term, should pass empty list. See: https://github.com/monarch-initiative/omim/issues/129
-        abbrev: Union[str, None] = None if not pref_symbols else pref_symbols[0]
+        pref_abbrev: Union[str, None] = None if not pref_symbols else pref_symbols[0]
```

Comment:
Let's talk about this one

Reply:
Sure thing, I added to the agenda.

I wonder if you are thinking about:
- #129

That is, the fact that all of the abbreviations should be passed to `cleanup_label()`.

Reply:
It was just this item, ` pref_symbols[0]` that I want to talk about.

Reply:
Roger!

Any concern over that line will likely be eventually addressed by:
- #153

However, that PR introduces big changes, so I'm not sure it's something we can merge without thorough review and maybe consulting others.

Reply:
We went over this in meeting. The code is just a little confusing because `prev_abbrev` has a few different uses in the code. However, this will go away and code will be easier to understand (and also more correct) whenever #153 is merged.